### PR TITLE
Add histogram selection + time info for torque time-series widget

### PR DESCRIPTION
**Pull request by viddo**
Waypoint for #786

torque-range3

This PR adds:
- Torque: animation for a selected time-series range
- Torque: format time-marker step info (time+date)

Basically just minor view changes, although had to modify the torque layer model slightly to handle events properly (for steps/range changes. 

Also notice that this PR depends on having a new minor version of torque.js, these changes are using https://github.com/CartoDB/torque/pull/243 for now I'm testing with a local checkout of that repo and using `npm link` to verify the changes working for this branch.

@javierarce can you review?
cc @javisantana for torque layer model changes

### CODE REVIEW

**Comment by alonsogarciapablo** [comment]
It looks really cool!

**Comment by viddo** [comment]
[Torque.js 2.14.0 was just released](https://github.com/CartoDB/torque/releases/tag/2.14.0), with updated dependency version the tests pass as expected too :tada:

File: src/geo/ui/widgets/time-series/torque-controls-view.js [hunk 1]
```diff
@@ -16,18 +15,13 @@ module.exports = View.extend({
...
-      );
-      this.show();
-    } else {
-      this.hide();
-    }
+    this.$el.html(
+      template({
+        label: this.model.get('isRunning')
+          ? '❚❚'
+          : '▶'
```
Comment (javierarce) [inline src/geo/ui/widgets/time-series/torque-controls-view.js]: :D

**Comment by javierarce** [comment]
:+1: awesome job

File: src/geo/ui/widgets/time-series/torque-controls-view.js (hunk 1 above)
Comment (viddo) [inline src/geo/ui/widgets/time-series/torque-controls-view.js]: I'll make the real icons in the next PR as part of doing the CSS etc. :wink:

Comment (javierarce) [inline src/geo/ui/widgets/time-series/torque-controls-view.js]: oh, no problem, I just loved the temporary idea :)

**Pull request merged**